actions/workflows/promote_package.yaml

```diff
@@ -0,0 +1,100 @@
+---
+version: '2.0'
+
+st2cd.promote_package:
+    description: Promote a package to production.
+    type: direct
+    input:
+      - package
+      - version
+      - revision
+      - os
+      - repo
+    tasks:
+
+      init:
+        action: core.local
```

Comment:
Hmm, we may want to run this as `core.remote` on one of the build hosts. Generally, we avoid using `st2build002` or whatever is the "build server" as the execution environment of any commands. Not always possible ofcourse.